### ISSUE TITLE
Check when fields have missing values

### ISSUE THREAD

**Issue by ghtmtt** [ghtmtt/DataPlotly#44]
Check the behavior and avoid Stack Traces

**Comment by ghtmtt** [comment]
cleaning function prototype:

```python
def cleanData(x, y):
    
    f1 = []
    f2 = []
    
    for i, j in enumerate(x):
        if x[i] and y[i] != NULL:
            f1.append(x[i])
            f2.append(y[i])

    return f1, f2
```

**Comment by ghtmtt** [comment]
function created in 2eb8379. Some tests made, but more tests needed.

**Comment by ghtmtt** [comment]
b08cfe3ff12b88c00c78e71df1eed1d314b6f372 should fix this issue (also related with #49)
